cmd/create_prs/create_prs.go

```diff
@@ -68,6 +71,15 @@ func run(c *cobra.Command, _ []string) {
 		readCampaignActivity.EndWithFailure(err)
 		return
 	}
+	prDescriptionUnchanged, err := campaign.PrDescriptionUnchanged(dir)
```

Comment:
Could we perhaps rename the function to `PrDescriptionUnchanged`? Unless there is any precedent in other boolean-returning functions?